steelpernach.prefab

```diff
@@ -0,0 +1,23 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:SteelDagger",
```

Comment:
@mrigendraagrawal Could you replace this with an icon of a Steel Pernach? Remember that the image must be compliant with the Apache 2.0 license.